mitosheet/src/components/taskpanes/UserDefinedImport/UserDefinedImportTaskpane.tsx

```diff
@@ -109,6 +136,60 @@ const UserDefinedImportTaskpane = (props: UserDefinedImportTaskpaneProps): JSX.E
                                 </Select>
                             </Col>
                         </Row>
+                        {userDefinedImporter?.parameters !== undefined && Object.keys(userDefinedImporter?.parameters).length > 0 && 
+                            <Row justify='start' align='center' title='TODO'>
+                                <Col>
+                                    <p className='text-header-3'>
+                                        Params
```

Comment:
Maybe change the text to `Set Param Values` and then have ? that explains what this section is. The end users are probably going to not really understand this.